frontend/src/components/Pages/FrontPage/MissionOverview/SafeZoneBanner.tsx

```diff
@@ -0,0 +1,39 @@
+import { Icon, Typography } from '@equinor/eds-core-react'
+import { tokens } from '@equinor/eds-tokens'
+import { AlertCategory } from 'components/Alerts/AlertsBanner'
+import { useLanguageContext } from 'components/Contexts/LanguageContext'
+import styled from 'styled-components'
+
+const StyledIcon = styled(Icon)`
+    width: 24px;
+    height: 24px;
+    color: ${tokens.colors.text.static_icons__default.hex};
+`
+const StyledLeftContent = styled.div`
+    display: flex;
+    padding-left: 20px;
+    flex-direction: row;
+    align-items: center;
+    gap: 10px;
+`
+
+interface SafeZoneBannerProps {
+    alertCategory: AlertCategory
+}
+
+export const SafeZoneBanner = ({ alertCategory }: SafeZoneBannerProps): JSX.Element => {
```

Comment:
Style of safe zone Banner and failed missions banner should be more similar
![image](https://github.com/equinor/flotilla/assets/33518988/4d0393ae-9ac3-4390-bdc7-0cd709f4f5e2)

Reply:
Should also center the safe zone banner content

Reply:
Have fixed this